pkg/doc/verifiable/credential_jwt.go

```diff
@@ -0,0 +1,205 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifiable
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/square/go-jose/v3"
+	"github.com/square/go-jose/v3/jwt"
+)
+
+// CredentialJWT is JWT Claims extension by Verifiable Credential.
+type CredentialJWT struct {
+	*jwt.Claims
+
+	Credential *rawCredential `json:"vc,omitempty"`
+}
+
+// Signed creates signed JWT (JWS)
+func (vcc *CredentialJWT) Signed(signatureAlg JWTAlgorithm, privateKey interface{}, keyID string) (string, error) {
```

Comment:
I’m not sure about this method name (or this particular abstraction). I see jwt has the same method name with a different signature.

I.e I think a Signed method would take a Signer.

Reply:
Fixed in a new version where this workaround is not used anymore.

Reply:
Maybe `JWS()` would sound better here?